spec/models/recruitment_cycle_spec.rb

```diff
@@ -2,15 +2,27 @@
 
 RSpec.describe RecruitmentCycle do
   describe '.current_year' do
-    it 'is 2020 before the end of cycle' do
+    it 'delegates to EndOfCycleTimetable' do
+      allow(EndOfCycleTimetable).to receive(:current_year)
+
+      RecruitmentCycle.current_year
+
+      expect(EndOfCycleTimetable).to have_received(:current_year)
+    end
+  end
+
+  describe '.next_year' do
+    it 'is 2021' do
```

Comment:
Could maybe add a context block or some description here, something along the lines of:

`it 'is 2021 if we're in the middle of the 2020 cycle'`

Reply:
Yep good idea (this and other comments re: context)